### ISSUE TITLE
Feature: flag help

### ISSUE THREAD

**Issue by mooshbeef** [Pocketkid2/Survival-Games#5]
Basicly, it would be nice to have a quick ingame lookup/cheatsheet for the flags you can set on the arena. Something like **/sg help flags** and **/sg help flag <FLAGNAME>**.

Probably best to aim for a 1 line per item list.

**Comment by Pocketkid2** [comment]
Yeah I know that the flag system in the plugin is currently a piece of crap. Give me some time and I'll work on a better one
